packages/tools/replay-tool/src/index.ts

```diff
@@ -8,6 +8,6 @@ export {
 	getNormalizedFileSnapshot,
 	loadContainer,
 	uploadSummary,
-} from "./helpers";
-export { ReplayArgs } from "./replayArgs";
-export { ReplayTool } from "./replayMessages";
+} from "./helpers.js";
```

Comment:
Maybe out of scope for this PR but are all of these exports really needed?

Reply:
just checked - all the exports are used.

Reply:
Thanks for the due diligence!